trl/trainer/online_dpo_trainer.py

```diff
@@ -233,6 +236,9 @@ def __init__(
             preprocess_logits_for_metrics=preprocess_logits_for_metrics,
         )
 
+        # Add dynamic parameter callback for beta
+        self.add_callback(DynamicParameterCallback("beta", args.beta))
```

Comment:
wdyt of this:

```suggestion
        if isinstance(args.beta, list):
            self.beta = args.beta[0]
            self.add_callback(DynamicParameterCallback("beta", args.beta))
        else:
            self.beta = args.beta
```

And remove the possibility in `DynamicParameterCallback` for `param_values` not to be a sequence?